go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/kubearmor/kubearmor-client
+
+go 1.16
+
+replace github.com/kubearmor/KubeArmor/pkg/KubeArmorPolicy => github.com/daemon1024/KubeArmor/pkg/KubeArmorPolicy v0.0.0-20210909152203-f9d3cb0a82e2
```

Comment:
We will not need this replacement after https://github.com/kubearmor/KubeArmor/pull/343